app/scripts/controllers/transactions/index.js

```diff
@@ -1708,7 +1708,7 @@ export default class TransactionController extends EventEmitter {
       // If the user rejects a transaction, finalize the transaction added
       // event fragment. with the abandoned flag set.
       case TRANSACTION_EVENTS.REJECTED:
-        id = `transaction-added-${txMeta.id}`;
+        id = `transaction-rejected-${txMeta.id}`;
```

Comment:
This is incorrect... it should be finalizing the transaction-added fragment. THere is no transaction-rejected fragment.

Reply:
I see, let me revert this change